File: 0344-reverse-string/0344-reverse-string.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {
        
        
	int l = 0, r = (int)s.size()-1;

	while(l < r){
		char temp = s[r];
		s[r] = s[l];
		s[l] = temp;
		l++;
		r--;
	}

        
    }
};

File: 0344-reverse-string/0344-reverse-string.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        l, r = 0, len(s)-1
        while l < r:
            temp_char = s[r]
            s[r] = s[l]
            s[l] = temp_char
            l += 1
            r -= 1
